core/templates/dev/head/pages/admin-page/config-tab/admin-config-tab.directive.ts

```diff
@@ -58,7 +57,7 @@ angular.module('oppia').directive('adminConfigTab', [
             return;
           }
 
-          $http.post(ADMIN_HANDLER_URL, {
+          AdminConfigTabBackendApiService.revertConfigProperty({
```

Comment:
It doesn't really make sense to pass in the action here, given it's the same as the call. I suggest refactoring to revertConfigProperty(configPropertyId) and letting the backend API service deal with the extra payload assembly needed to actually communicate with the backend.

Ditto below.

Reply:
done